src/client-scopes/add/components/ScriptComponent.tsx

```diff
@@ -20,7 +20,20 @@ export const ScriptComponent = ({
     <FormGroup
       label={t(label!)}
       labelIcon={
-        <HelpItem helpText={t(helpText!)} forLabel={t(label!)} forID={name!} />
+        <HelpItem
+          helpText={
+            <span>
+              {helpText?.split("\n").map((item, key) => (
+                <Fragment key={key}>
```

Comment:
Instead of using the index as a key perhaps you can use the item itself?
```suggestion
                <Fragment key={item}>
```

Reply:
Doesn't really matter as it's a static list

Reply:
True, still better not to do it out of convention. I'd like to add an ESLint rule for this at some point as well.